lib/maplibre-compose-material3/src/androidMain/kotlin/dev/sargunv/maplibrecompose/material3/util.android.kt

```diff
@@ -0,0 +1,5 @@
+package dev.sargunv.maplibrecompose.material3
+
+import dev.sargunv.maplibrecompose.material3.controls.ScaleBarMeasure
+
+internal actual fun scaleBareMeasurePreference(): ScaleBarMeasure? = null
```

Comment:
API 28" has `LocaleData.getMeasurementSystem`: https://developer.android.com/reference/android/icu/util/LocaleData#getMeasurementSystem(android.icu.util.ULocale)

not quite a "preference" since I think the user can't set it, but still probably good to rely on that to inform the default on APIs where it's available.

And then the big hardcoded list can be commented with a note for removal when minSdk >= 28

Reply:
I've found the data source for the CLDR data on the preferred measurement system per country. It is quite coarse, much more coarse than what I put in the code:

https://github.com/unicode-org/cldr/blob/main/common/supplemental/units.xml#L478-L496

The gist of that data: Use feet+miles in `US`, use yard+miles in `GB`. So, it's worse than the data I researched.

Reply:
> And then the big hardcoded list

(By the way, it is only big because this "spotless check format" insists on a linebreak after each country and language code :-/)

Reply:
Hmm, with my other comment on "primary" and "secondary", maybe we use this data for the default "primary", then:
* for locales with primary = US or UK, secondary default is SI
* for locales with primary = SI, there's a hardcoded list of secondary=US or UK, rest are null